docs/sources/breaking-changes/breaking-changes-v10-0.md

```diff
@@ -343,7 +343,7 @@ Here are some of the benefits of create-plugin:
 
 - **Improved testing capabilities:** Testing plugins with @grafana/create-plugin is much easier with GitHub workflows that automate unit and e2e test runs whenever changes are pushed to GitHub.
 
-- **Better documentation:** The [documentation](https://grafana.github.io/plugin-tools/docs/creating-a-plugin) for @grafana/create-plugin is more comprehensive and easier to discover than that of @grafana/toolkit.
+- **Better documentation:** The [documentation](https://grafana.github.io/plugin-tools/docs/getting-started/creating-a-plugin) for @grafana/create-plugin is more comprehensive and easier to discover than that of @grafana/toolkit.
```

Comment:
```suggestion
- **Better documentation:** The [documentation](https://grafana.github.io/plugin-tools/docs/getting-started/) for @grafana/create-plugin is more comprehensive and easier to discover than that of @grafana/toolkit.
```

Reply:
Commit https://github.com/grafana/plugin-tools/pull/320/commits/9506612fb313bcdf7140a6e187e5ab261a223c87 will delete that file. I think we want the main URL anyway.

Reply:
@josmperez there's a lot more links to that file, I'll fix them all again once we've landed your PR